docs/code/round_2/pages.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-import re
 from bok_choy.page_object import PageObject
+import re
```

Comment:
I'm not sure why you feel this is necessary. This reordering violates our style conventions: https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order